src/SMTPClient.jl

```diff
@@ -4,9 +4,9 @@ module SMTPClient
 
 using LibCURL
 
-import Base: convert, send, show
+import Base: convert, show
 
-export SendOptions, SendResponse
+export SendOptions, SendResponse, send
```

Comment:
Well, the `send` function is moved to the stdlib `Sockets`.
So I think import it is enougth.
```julia
import Sockets: send
```

Reply:
At least for me that does fail with:
RROR: LoadError: UndefVarError: send not defined
Stacktrace:
 [1] top-level scope at none:0
in expression starting at /home/arch/Programming_Documents/Julia_projects/mail test.jl:8

Which does mean that the example on your readme page does not work properly because the send function is not given over to julia.

Reply:
ERROR: LoadError: LibCURL.curl_easy_perform() failed: Failed sending data to the peer
Stacktrace:
 [1] _do_send(::String, ::Array{String,1}, ::String, ::SendOptions, ::SMTPClient.ReadData) at /home/ricks/.julia/packages/SMTPClient/IWLdX/src/SMTPClient.jl:141
 [2] send(::String, ::Array{String,1}, ::String, ::Base.GenericIOBuffer{Array{UInt8,1}}, ::SendOptions) at /home/arch/.julia/packages/SMTPClient/IWLdX/src/SMTPClient.jl:233
 [3] top-level scope at none:0
in expression starting at /home/arch/Programming_Documents/Julia_projects/mail test.jl:8

Reply:
I am getting this when I add the Import socket but leave the export send. Because you need to export send to the program that wants to call smtpclient.

Reply:
oh, let me check it. seems we need export it as well.

Reply:
I guess so but there is another error I am experiencing when exporting

Reply:
ok, exporting it is correct. and the error message is caused by another issue.

Reply:
So importing Sockets: send an then further investigation ok